src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/DataAccess.cs

```diff
@@ -23,12 +23,21 @@ public Account GetAccountByLegalEntityId(long legalEntityId)
             return account;
         }
 
+        public Account GetAccountByAccountLegalEntityId(long accountId, long accountLegalEntityId)
+        {
+            using var dbConnection = new SqlConnection(_connectionString);
+            var account = dbConnection.QuerySingle<Account>("SELECT * FROM Accounts WHERE Id = @accountId AND AccountLegalEntityId = @AccountLegalEntityId",
+                new { accountId, accountLegalEntityId });
+
+            return account;
+        }
+
         public void SetupAccount(Account account)
```

Comment:
Can you use Dapper contrib as in EI-49?  It's a lot simpler I think.

Reply:
hmm this is a test code , can refactor once EI-49 is merged?